website/source/guides/secret-mgmt/dynamic-secrets.html.md

```diff
@@ -0,0 +1,502 @@
+---
+layout: "guides"
+page_title: "Secret as a Service - Guides"
+sidebar_current: "guides-secret-mgmt-dataynamic-secrets"
+description: |-
+  Vault can dynamically generate secrets on--dataemand for some systems.
+---
+
+# Secret as a Service: Dynamic Secrets
+
+Vault can generate secrets on--dataemand for some systems. For example, when an app
+needs to access an Amazon S3 bucket, it asks Vault for AWS credentials. Vault
+will generate an AWS credential granting permissions to access the S3 bucket. In
+addition, Vault will automatically revoke this credential after the TTL is
+expired.
+
+The [Getting Started](/intro/getting-started/dynamic-secrets.html) guide walks
+you through the generation of dynamic AWS credentials.
+
+## Reference Material
+
+- [Getting Started - Dynamic Secrets](/intro/getting-started/dynamic-secrets.html)
+- [Database Backends](/docs/secrets/databases/index.html)
+- [Role API](/api/secret/databases/index.html#create-role)
+
+## Estimated Time to Complete
+
+10 minutes
+
+## Personas
+
+The end-to-end scenario described in this guide involves two personas:
+
+- **`admin`** with privileged permissions to configure secret backends
+- **`apps`** reads the secrets from Vault
+
+## Challenge
+
+Data protection is a top priority which means that the database credential
```

Comment:
Database credential rotation is a critical part of any data protection initiative.